### PR TITLE
EUI-446 Pagination implemented in search results

### PR DESCRIPTION
**Pull request by praveenrajendran101**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: src/cases/containers/case-search/case-search.component.ts [hunk 1]
```diff
@@ -1,7 +1,7 @@
 import { Component, OnInit, ViewEncapsulation, OnDestroy } from '@angular/core';
 import { Store, select } from '@ngrx/store';
 import * as fromCasesFeature from '../../store';
-import { Jurisdiction, CaseType, CaseState, SearchResultView, PaginationMetadata } from '@hmcts/ccd-case-ui-toolkit';
+import { Jurisdiction, CaseType, CaseState, SearchResultView, PaginationMetadata, SearchResultViewItem } from '@hmcts/ccd-case-ui-toolkit';
```
Comment (adnanakgun) [inline src/cases/containers/case-search/case-search.component.ts]: If the "SearchResultViewItem" import is new code, is it being used in this file? If not please remove it.

File: src/cases/containers/case-search/case-search.component.ts [hunk 2]
```diff
@@ -87,14 +90,19 @@ export class CaseSearchComponent implements OnInit, OnDestroy {
         ...result[3]
       };
     });
-
+    this.paginationMetadata$.subscribe(result => {
```
Comment (adnanakgun) [inline src/cases/containers/case-search/case-search.component.ts]: Create a subscription for this and unsubscribe from it in the OnDestroy please.

File: src/cases/containers/case-search/case-search.component.ts [hunk 3]
```diff
@@ -87,14 +90,19 @@ export class CaseSearchComponent implements OnInit, OnDestroy {
...
+      }
+    });
     this.resultSubscription = this.resultView$.subscribe(resultView => {
-
-      this.paginationMetadata = new PaginationMetadata();
-      const resultViewResultsLength = resultView.results ? resultView.results.length : 0;
-      this.paginationMetadata.total_results_count = resultViewResultsLength;
-      this.paginationMetadata.total_pages_count = Math.ceil(resultViewResultsLength / this.paginationSize);
       this.resultsArr = resultView.results;
+      console.log(this.paginationMetadata.total_pages_count);
```
Comment (adnanakgun) [inline src/cases/containers/case-search/case-search.component.ts]: Remove this log message please

File: src/cases/containers/case-search/case-search.component.ts (hunk 1 above)
Comment (praveenrajendran101) [reply src/cases/containers/case-search/case-search.component.ts]: done

**Pull request merged**